interfaces/builtin/multipass.go

```diff
@@ -0,0 +1,51 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2018 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const multipassSummary = `allows access to the Multipass socket`
+
+const multipassBaseDeclarationSlots = `
+  multipass:
```

Comment:
Can you please comment on the design of the policy? Why cannot the multipass interface be installed or connected?

Reply:
The intent is for Multipass to only provide a stable command line interface. 

We don't want to expose the Multipass server socket to other snaps that might try ship their own multipass client binary, as the binary protocol can and will change.

I believe we can achieve this by preventing the multipass interface from being installed or connected.

Reply:
In addition to the reasons @gerboland gave, we want to mimic the lxd and docker interfaces since access to this socket gives quite privileged access so IMO the base declaration is correctly setup here.

Reply:
After another pass, this client interface was completely removed, as the socket is now kept in $SNAP_COMMON